pkg/controllers/proxyconfig/proxyconfig_controller.go

```diff
@@ -52,11 +54,12 @@ func NewProxyConfigChecker(
 		WithInformers(
 			routeInformer.Informer(),
 		).
-		ResyncEvery(5 * time.Minute).
+		ResyncEvery(60 * time.Minute).
 		WithSyncDegradedOnError(operatorClient)
 
-	for ns := range caConfigMaps {
-		c.WithInformers(
```

Comment:
separate these changes to the first commit, the lazy checking goes to a second one

Reply:
good idea, done